common/postgresql/values.yaml

```diff
@@ -16,7 +16,7 @@ global:
       max_client_conn: 16384
       reserve_pool_size: 100
       min_pool_size: 1
-      default_pool_size: 50 # per user and database
+      default_pool_size: 100 # per user and database
```

Comment:
If I read it correctly, you set the pool size higher than the number of db connections (one line below)

Reply:
ok, i thought this is why we have a pooler. pool connections > db connections.
actually it is: db_count+user_count*(default_pool_size + reserve_pool_size, max_db_connections)